picard/util/imagelist.py

```diff
@@ -35,48 +35,51 @@ def __getitem__(self, k):
             return result
 
 
+class State:
```

Comment:
I'd prefer a longer name here, may be `ImageListState`, `State` is too general for such specific class